### ISSUE TITLE
Feature presentation pattern

### ISSUE THREAD

**Issue by StoneCypher** [StoneCypher/fsl#1127]
Once the feature comparison matrix is in place, we need to write docs on each major feature one by one, then link them from the main readme.  In order to do that well, we should create a GoF-style `"this is what a feature description looks like"` pattern.  Might as well do it in-place in this issue.

Changes to the pattern should be written in comments, and will be merged into the issue body selectively as an accept.  Pattern begins below the HR.

Each major feature (by which we mean things like "states," "transitions," "actions," "hooks," etc, but not variants like kinds of transition or kinds of action) will get its own sidebar page.  This pattern language governs the minimum for those pages.

Draft follows below.  Criticism and extension are both requested.  Ideal is a "not-too-large" end result.

I've never tried tagging people on an open source project before.  Let's see how this goes.  @machinshin @blackeuler @msmorgan @forestbelton @cavanstone @xtovski

----

# Feature title

Brief (two-sentence max) non-exhaustive description.

```fsl
example -> simple -> implementation;
```

Detailed description.  

Explanation of why this is needed, and what we'd have to do without it.

Explanation of which kinds of machines need this, and which don't, if practical.

At least three significantly different practical use case examples, if possible.  At least one of them written out in code, with a graph.

```fsl
big -> [complex practical interesting] => real_use_case;
```

Links to relevant manual points.

Other names this might go under in other machines; examples where practical.

**Comment by StoneCypher** [comment]
Superseded — folded into the FSL manual / taxonomy / curriculum program. Closing per the approved triage ledger (`notes/superpowers/eras/dispositions/fsl-triage.md`).
